addons/project/report/project_report.py

```diff
@@ -10,21 +10,25 @@ class ReportProjectTaskUser(models.Model):
     _order = 'name desc, project_id'
     _auto = False
 
-    name = fields.Char(string='Task Title', readonly=True)
+    name = fields.Char(string='Task', readonly=True)
     user_id = fields.Many2one('res.users', string='Assigned To', readonly=True)
+    create_date = fields.Datetime("Create Date", readonly=True)
     date_assign = fields.Datetime(string='Assignment Date', readonly=True)
     date_end = fields.Datetime(string='Ending Date', readonly=True)
     date_deadline = fields.Date(string='Deadline', readonly=True)
     date_last_stage_update = fields.Datetime(string='Last Stage Update', readonly=True)
     project_id = fields.Many2one('project.project', string='Project', readonly=True)
-    working_days_close = fields.Float(string='# Working Days to Close',
+    working_days_close = fields.Float(string='Working Days to Close',
         digits=(16,2), readonly=True, group_operator="avg",
         help="Number of Working Days to close the task")
-    working_days_open = fields.Float(string='# Working Days to Assign',
+    working_days_open = fields.Float(string='Working Days to Assign',
         digits=(16,2), readonly=True, group_operator="avg",
         help="Number of Working Days to open the task")
-    delay_endings_days = fields.Float(string='# Days to Deadline', digits=(16,2), readonly=True)
+    delay_endings_days = fields.Float(string='Days to Deadline', digits=(16, 2), group_operator="avg", readonly=True)
     nbr = fields.Integer('# of Tasks', readonly=True)  # TDE FIXME master: rename into nbr_tasks
+    working_hours_open = fields.Float(string='Working Hours to Assign', digits=(16, 2), readonly=True, group_operator="avg", help="Number of Working Hours to open the task")
+    working_hours_close = fields.Float(string='Working Hours to Close', digits=(16, 2), readonly=True, group_operator="avg", help="Number of Working Hours to close the task")
+    rating_last_value = fields.Float('Rating Value (/5)', group_operator="avg", readonly=True, groups="project.group_project_rating")
     priority = fields.Selection([
```

Comment:
I am wondering why we still have High in the report as it is no more available in project.task. I found out this issue thank to the domain in the filter [here](https://github.com/odoo/odoo/pull/72587/files#diff-2cd049c7f1399ab6455f42f422cf21591cf40fbe78d5c067870e2b31c3e3716bR44)

Reply:
@IT-Ideas Yeah, You are right. I have changed it.Thanks :)